realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -392,6 +392,10 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    public void removeAccessToken (URI serverURL) {
```

Comment:
So this one depends on #4040 ? You will have conflicts here after that merged. Just a reminder :)